charts/federation-v2/README.md

```diff
@@ -29,24 +29,37 @@ group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
 
 ## Installing the Chart
 
-First you'll need to create the reserved namespace for registering clusters with the
-cluster registry:
+First you'll need to create the reserved namespace for registering clusters with the cluster registry:
 
 ```bash
 $ kubectl create ns kube-multicluster-public
 ```
 
-Install the chart with the release name `federation-v2`:
+Next, add the chart to your local resitory.
+```bash
+$ helm repo add federation-v2 https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/master/charts/federation-v2
```

Comment:
Would it make sense to suggest adding it as `fedv2-repo` or something that differentiates from the name of the chart?

Reply:
Yes, and it would open room for any new charts this repo would have in the future. Working with #754 seems to work perfectly. 

https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/add-chart-repo-index/charts/

This path assumes a new branch named "add-chart-repo-index" is going to be made. Can you confirm this?